src/gurt/telemetry.c

```diff
@@ -81,13 +82,16 @@ static struct d_tm_shmem {
 	uint32_t                 retain : 1, /** retain shmem region during exit */
 	    sync_access                 : 1, /** enable sync access to shmem */
 	    retain_non_empty            : 1, /** retain shmem region if it is not empty */
-	    multiple_writer_lock        : 1; /** lock for multiple writer */
+	    multiple_writer_lock        : 1, /** lock for multiple writer */
+	    use_non_shared_mem          : 1; /** use non-shared memory for allocs */
 	int			 id; /** Instance ID */
 } tm_shmem;
 
 /* Internal helper functions */
-static int allocate_shared_memory(int srv_idx, size_t mem_size,
-				  struct d_tm_shmem_hdr **shmem);
+static int
+allocate_memory_segment(int srv_idx, size_t mem_size, bool shared, struct d_tm_shmem_hdr **shmem);
+static void
+			   release_ctx_mem(struct d_tm_context **ctx, bool destroy_shmem);
```

Comment:
guessing clang-format did this... I know it can do some funky indentation, but any idea why the first function isn't indented but the second is?

Reply:
Dunno... I'll see if I can make it behave.